xarray/tests/test_sparse.py

```diff
@@ -1,889 +0,0 @@
-import pickle
```

Comment:
I don't think the removal of this file was intentional.

Reply:
I restored this file when merging master into my branch.